app/assets/javascripts/static.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Application entry point.
+ */
+require([
+  'jquery',
+  'underscore',
+  'Class',
+  'backbone',
+  'views/AboutView'
+], function($, _, Class, Backbone, AboutView) {
+
+  'use strict';
+
+  var Static = Class.extend({
+
+    $el: $('body'),
+
+    init: function() {
+      this._initViews();
+    },
+
+    /**
+     * Initialize the map by starting the history.
+     */
+    _initApp: function() {
+      if (!Backbone.History.started) {
+        Backbone.history.start({pushState: true});
+      }
+    },
+
+    /**
+     * Initialize Application Views.
+     * CAUTION: Don't change the order of initanciations if
+     * you are not completely sure.
+     */
+    _initViews: function() {
+      new AboutView();
```

Comment:
Yes you can but for me right now is perfect because the AboutView takes care of the download events and the general stuff and the SideBarNavView only takes care of the tabs.

So at the end would be easier to add the event in the SideBarNavView but if tomorrow the links are outside this layout we will have to change it again.